deps.windows/70-libdatachannel.ps1

```diff
@@ -0,0 +1,84 @@
+param(
+    [string] $Name = 'libdatachannel',
+    [string] $Version = '0.19.0-alpha.4',
+    [string] $Uri = 'https://github.com/paullouisageneau/libdatachannel.git',
+    [string] $Hash = '709a66339451bb4c8d4e5ced78c67605ec09da31',
+    [array] $Targets = @('x64'),
```

Comment:
Is this restricted to x64 for a specific reason?

Reply:
thought we did not support x86 anymore on win ? you're thinking arm64 ? i hope not

Reply:
oh https://github.com/obsproject/obs-deps/pull/196
😮

Reply:
Dropping x86 comes in #196, as you've noted.

Reply:
so what do you recommend ? enable both for x86 & x64 until Pat's PR lands or if it's likely Pat's PR will land before this one, then stick to x64 ?
I guess it doesn't matter much then

Reply:
Simply don't include this line. It will either be added as part of #196, or will need to be added if that is merged first.

Reply:
done